spinalcordtoolbox/reports/assets/index.html

```diff
@@ -262,7 +290,63 @@
         }
     }
 
-    <!-- Toggle overlay -->
+    function download_yaml(marker) {
+        let res = {};
+        // TODO: Coordinate with PR #3320 to add entries for sct_dmri_moco/sct_fmri_moco.
```

Comment:
Now that #3320 has been merged, I'll quickly address this TODO before this is merged. :slightly_smiling_face:

Reply:
Fixed by 86ec6a2. 

I chose to use a new image type marker, `FILES_TIMESERIES`, because the dmri/fmri images are 4D and the QC displays gifs (which is different than any other type of QC report).